skills/declarative/calendarSkill/dialogs/CreateDialog/CreateDialog.dialog

```diff
@@ -0,0 +1,1122 @@
+            "id": "AIp7JR"
+          },
+          "name": "CreateDialog.OnIntent_RemoveTeamsLink"
+        },
+        {
+          "$kind": "Microsoft.EmitEvent",
+          "$designer": {
+            "id": "dLCdSc"
+          },
+          "eventName": "SetOnlineMeeting",
+          "eventValue": {
+            "action": "remove"
+          }
+        }
+      ]
+    },
+    {
+      "$kind": "Microsoft.OnIntent",
+      "$designer": {
+        "id": "SuPEFo",
+        "name": "OnIntent (SetLocation)"
+      },
+      "intent": "SetLocation",
+      "actions": [
+        {
+          "$kind": "Microsoft.TraceActivity",
+          "$designer": {
+            "id": "xAorBS"
+          },
+          "name": "CreateDialog.OnIntent_SetLocation"
+        },
+        {
+          "$kind": "Microsoft.EmitEvent",
+          "$designer": {
+            "id": "R40YIi"
+          },
+          "eventName": "SetLocation"
+        }
+      ]
+    },
+    {
+      "$kind": "Microsoft.OnIntent",
+      "$designer": {
+        "id": "9KQ0ec",
+        "name": "OnIntent (SetDescription)"
+      },
+      "intent": "SetDescription",
+      "actions": [
+        {
+          "$kind": "Microsoft.TraceActivity",
+          "$designer": {
+            "id": "09ybf4"
+          },
+          "name": "CreateDialog.OnIntent_SetDescription"
+        },
+        {
+          "$kind": "Microsoft.EmitEvent",
+          "$designer": {
+            "id": "8Pwzv9"
+          },
+          "eventName": "SetDescription"
+        }
+      ]
+    },
+    {
+      "$kind": "Microsoft.OnIntent",
+      "$designer": {
+        "id": "aFeCU4",
+        "name": "OnIntent (Skip)"
+      },
+      "intent": "Skip",
+      "actions": [
+        {
+          "$kind": "Microsoft.TraceActivity",
+          "$designer": {
+            "id": "Y5PPQB"
+          },
+          "name": "CreateDialog.OnIntent_Skip"
+        },
+        {
+          "$kind": "Microsoft.SetProperty",
+          "$designer": {
+            "id": "L2LKtF"
+          },
+          "property": "turn.skip",
+          "value": "=true"
+        }
+      ]
+    },
+    {
+      "$kind": "Microsoft.OnIntent",
+      "$designer": {
+        "id": "8caScR",
+        "name": "OnIntent (Cancel)"
+      },
+      "intent": "Cancel",
+      "actions": [
+        {
+          "$kind": "Microsoft.TraceActivity",
+          "$designer": {
+            "id": "lnSf08"
+          },
+          "name": "CreateDialog.OnIntent_Cancel"
+        },
+        {
+          "$kind": "Microsoft.SendActivity",
+          "$designer": {
+            "id": "luOnHv"
+          },
+          "activity": "${SendActivity_luOnHv()}"
+        },
+        {
+          "$kind": "Microsoft.CancelAllDialogs",
+          "$designer": {
+            "id": "dSHOIO"
+          },
+          "activityProcessed": true
+        }
+      ]
+    }
+  ],
+  "generator": "CreateDialog.lg",
+  "recognizer": "CreateDialog.lu.qna",
+  "id": "CreateDialog"
+}
```

Comment:
It should also have a OnIntent (SetDuartion) to support change duration in create flow.